docs/getting-started/cluster-setup.md

```diff
@@ -0,0 +1,206 @@
+# Setting Up a Cluster
+You can setup a cluster in [Kubernetes](http://kubernetes.io/docs/whatisk8s/) or Jenkins.
+
+## Setting Up a Cluster on AWS using Kubernetes
+
+### Prerequisites
+- [kubectl](http://kubernetes.io/docs/user-guide/prereqs/)
+- an [AWS](http://aws.amazon.com) account
+- [AWS CLI](https://aws.amazon.com/cli/)
+- [Screwdriver tables on DynamoDB](datastore-setup/)
+
+### Create your cluster
```

Comment:
This whole document is called "Setting up a cluster". Is this referring to the entire cluster? Or just the kubernetes cluster?